modules/engage-theodul-plugin-tab-transcript/src/main/resources/static/utils.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
See if you need this file - not really like any of the other calls refer to any utility functions

Reply:
Need this file for the getTimeInMilliseconds function.